src/LocalField/LocalField.jl

```diff
@@ -134,6 +132,10 @@ end
 #
 ################################################################################
 
+function base_ring(L::LocalField)
```

Comment:
shouldn't this be `base_ring(L::LocalField) = base_field(L)` instead?
But `base_ring` was never (intended to be) part of any useful interface, it was just indicating, in recursively defined rings, the previous step. Strictly here, `base_ring` might even have been the polynomial ring...
`base_field`, `coefficient_ring/field` were intended to be more useful